ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel.hs

```diff
@@ -147,6 +157,41 @@ observableRollbackUseInTestsOnly :: PassiveWallet -> IO ()
 observableRollbackUseInTestsOnly PassiveWallet{..} =
     update' _wallets $ ObservableRollbackUseInTestsOnly
 
+{-------------------------------------------------------------------------------
+  Active wallet
+-------------------------------------------------------------------------------}
+
+-- | Initialize the active wallet
+activeWalletComponent
+    :: PassiveWallet
+    -> ComponentM ActiveWallet
+activeWalletComponent walletPassive = do
+    let logMsg = _walletLogMessage walletPassive
+    buildComponent "Active wallet"
+        (return ActiveWallet{..})
+        (\_ -> liftIO $ do
+            logMsg Error "stopping the wallet submission layer..."
+        )
+
+bracketActiveWallet
```

Comment:
Didn't we decide to put such `bracket*` functions to tests, because they are used only there?